packages/evm/src/precompiles/0e-bls12-g2mul.ts

```diff
@@ -13,8 +11,6 @@ const {
 } = require('./util/bls12_381')
 
 export async function precompile0e(opts: PrecompileInput): Promise<ExecResult> {
-  if (isFalsy(opts.data)) throw new Error('opts.data missing but required')
```

Comment:
Hmm, I already overlooked a similar case on some other PR, but since it is reoccuring here: I vaguely remember that we introduced this for the JS side of things, since there error messages are unreliable if some parameter is not passed in correctly and things are not guarded by type checking.

Not sure, eventually worth to re-add.

On the other hand, to admit, we do not really have a consistent JS guarding strategy throughout the monorepo anyhow.

Hmm.

Anhow: just wanted to at least mention.